src/framework/preferences/js/StarterSchemas.js

```diff
@@ -21,11 +21,41 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
      * contrast, table of contents, inputs larger and emphasize links
      *******************************************************************************/
 
+    var check;
+    var widthCheck = function() {
```

Comment:
I tried encapsulating the check in a function and the use a expander, but I got and error. For now I will leave it like that. If you have a better I idea you can try it, because I don't know which will be the best solution for this case.